### PR TITLE
0.4.0.10 Add PiClone functionality to Admin Console

### PR DESCRIPTION
**Pull request by tim-moody**
still needs plays to install piclone_cmd

### CODE REVIEW

**Comment by tim-moody** [comment]
passed functional testing on raspbian

**Comment by tim-moody** [comment]
uses piclone_cmd_aarch64 and piclone_cmd_armv7l from https://github.com/georgejhunt/piclone_cmd

**Comment by tim-moody** [comment]
expects the proper version for the rpi arch in /usr/bin

File: roles/cmdsrv/files/iiab-cmdsrv3.py [hunk 1]
```diff
@@ -2106,6 +2124,22 @@ def sync_menu_item_defs(cmd_info):
...
+        resp = cmd_error(cmd_info['cmd'], msg='Image copy only supported on Raspberry Pi at this time.')
+        return (resp)
+
+    dest_dev = cmd_info['cmd_args']['dest_dev']
+    comp_proc = adm.subproc_run('ls ' + dest_dev)
+    if comp_proc.returncode != 0:
+        resp = cmd_error(cmd_info['cmd'], msg='Device ' + dest_dev + ' not found.')
+        return (resp)
+
+    job_command = "/usr/Sbin/piclone_cmd " + dest_dev
```
Comment (jvonau) [inline roles/cmdsrv/files/iiab-cmdsrv3.py]: typo s/b /usr/sbin/piclone_cmd not /usr/Sbin/piclone_cmd

**Comment by tim-moody** [comment]
never change after testing

**Pull request merged**